### ISSUE TITLE
PickList: String values are encoded on submit

### ISSUE THREAD

**Issue by qnox** [primefaces/primefaces#4036]
1) Environment
- PrimeFaces version: 6.2.8
- Does it work on the newest released PrimeFaces version? Version? **No**
- Does it work on the newest sources in GitHub? (Build by source -> https://github.com/primefaces/primefaces/wiki/Building-From-Source) **No**
- Application server + version: JBoss EAP 7.0.8.GA
- Affected browsers: Not depends on the browser

## 2) Expected behavior
`DualModel<String>` containing strings with `&` is supposed to contain same string after submit

## 3) Actual behavior
`DualModel<String>` contains encoded strings with `&amp;` instead of `&` after submit

## 4) Steps to reproduce
Create picklist with `DualModel<String>` filled with strings with `&`

## 5) Sample XHTML
```xml
      <h:form>
        <p:pickList value="#{pickListModel.model}"
                    var="value"
                    itemLabel="#{value}"
                    itemValue="#{value}">
        </p:pickList>

        <p:commandButton value="Submit" action="#{pickListModel.post}" />
      </h:form>
```
## 6) Sample bean
```java
package org.example;

import java.util.ArrayList;
import java.util.Collections;
import javax.el.MethodExpression;
import javax.faces.view.ViewScoped;
import org.primefaces.model.DualListModel;
import org.springframework.stereotype.Component;

@Component
@ViewScoped
public class PickListModel {

    private DualListModel<String> model;

    public PickListModel() {
        model = new DualListModel<>();
        model.setSource(new ArrayList<>(Collections.singletonList("&source&")));
        model.setTarget(new ArrayList<>(Collections.singletonList("&target&")));
    }

    public DualListModel<String> getModel() {
        return model;
    }

    public void setModel(DualListModel<String> model) {
        this.model = model;
    }

    public void post() {
        System.out.println(model.getTarget());
    }
}
```

**Comment by Rapster** [comment]
It'd be great if you could analyse the problem and provide a PR :wink:

**Comment by melloware** [comment]
There needs to be 2 things to fix this issue:

1. A new property added to Picklist called "escape" like other components so you can say "escape=false" to turn off the HTML escaping.

2. In picklist.js this line needs to check that "escape" property and choose whether or not to call PrimeFaces.escapeHTML.
```javascript
escapedItemLabel = (itemLabel) ? PrimeFaces.escapeHTML(itemLabel) : ''
```

**Comment by melloware** [comment]
You are right.  I just checked in the fix.

**Comment by orange-buffalo** [comment]
Thanks for the improvement, guys! Is it possible to port it to the 6.2? We have Elite subscription and need this functionality.

**Comment by tandraschko** [comment]
thats up to @mertsincan

**Comment by orange-buffalo** [comment]
@mertsincan, is it possible to include this into the 6.2.x releases?

**Comment by mertsincan** [comment]
Thanks a lot for PR, @melloware ;)

**Comment by UlrichLohrmann** [comment]
The change is not backward compatible. We create PickList instance using Java code and if escape property is not set a NullPointerException will occur.
A call to pickList.setEscape(...) is required after upgrading to Primefaces version 6.2.14.

**Comment by mertsincan** [comment]
Hi @UlrichLohrmann,
Fixed for 6.2.15. https://github.com/primefaces/primefaces/issues/4413

**Comment by UlrichLohrmann** [comment]
Great, thank you.
